contracts/lib/FulfillmentApplier.sol

```diff
@@ -122,10 +122,12 @@ contract FulfillmentApplier is FulfillmentApplicationErrors {
                     .startAmount = (execution.item.amount -
                     considerationItem.amount);
             }
+
+            // Reuse consideration amount.
```

Comment:
One more nit here: since we're now more consistent across treatment of these two conditionals, can we update this comment to be more inline with the other one, i.e. `// Reduce total offer amount to equal the consideration amount.`